viewing_party/party.py

```diff
@@ -1,23 +1,415 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    """
+    Input: title (string), genre (string), rating (float)
+    Output: If parameters are truthy, return dictionary. If not, return None
+    """

```

Comment:
Nice job including docstrings!  

It's great to get in the habit for when you're working on larger projects.  :smiley: